ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -455,7 +454,7 @@ export function PreviewConcierge(props: any) {
             }
             getHostToGuestBus().next({
               type: HOST_CHECK_IN,
-              payload: { editMode: false, highlightMode }
+              payload: { editMode: false, highlightMode, rteConfig: rteConfig.pageBuilder ?? {} }
```

Comment:
I added this for the pageBuilder rte to read a setup with id `pageBuilder`, is that ok? or should this one read base?